pilot-app/src/components/form/fields/wimTextline.vue

```diff
@@ -0,0 +1,48 @@
+﻿<template>
+    <div :class="classname">
+        <label v-if="prefix(field)" v-html="prefix(field)"></label>
+        <label :class="getClass(field)"
+               :name="field.propertyName"
+               :id="field.propertyName"
+               v-html="value"></label>
+        <label v-if="suffix(field)" v-html="suffix(field)"></label>
+    </div>
+</template>
+<script>
+    import { fieldMixin } from './../form';
+
+    export default {
+        props: ['field', 'classname'],
+        mixins: [fieldMixin],
+        data() {
+            return {
+                value: "",
+            }
+        },
+        methods: {
+            getClass(field) {
+                let expression = '';//this.expressCell(field.expression);
+                if (typeof (field.className) !== 'undefined' && field.className) {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8170341)
```suggestion
                if (typeof (field.className) !=="undefined"' && field.className) {
```